src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1761,16 +1764,22 @@ private BaseListSyntax ParseBaseList()
                 // any additional types
                 while (true)
                 {
-                    if (this.CurrentToken.Kind == SyntaxKind.OpenBraceToken ||
-                        this.IsCurrentTokenWhereOfConstraintClause())
+                    if (AtEndOfBaseList())
                     {
                         break;
                     }
-                    else if (this.CurrentToken.Kind == SyntaxKind.CommaToken || this.IsPossibleType())
+                    else if (this.CurrentToken.Kind == SyntaxKind.CommaToken ||
+                             this.IsPossibleType())
```

Comment:
This is code style change. Wouldn't it be more appropriate to move that to separate commit?